### PR TITLE
circleci: add initial CI

### PR DESCRIPTION
**Pull request by iaguis**
This sets up a minikube cluster; installs helm, the service catalog, the
operator, and the broker; creates a redis service instance; and shows
running pods.

Tests should be the next step.

Signed-off-by: Iago López Galeiras <iago@kinvolk.io>

### CODE REVIEW

**Comment by iaguis** [comment]
PTAL

**Comment by iaguis** [comment]
Updated.

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+          command: |
+            until svcat get classes | grep nginx; do
+              sleep 1
+            done
+            kubectl apply -f manifests/redis/service-instance.yaml
+      - run:
+          name: Print pods and logs
+          command: |
+            kubectl get pods --all-namespaces
+            kubectl -n=habitat-broker logs -lapp=habitat-service-broker-habitat-service-broker --tail=100
```
Comment (asymmetric) [inline .circleci/config.yml]: The label looks weird. Is it really so?

Comment (iaguis) [reply .circleci/config.yml]: Unfortunately, yes. This is something we should fix.

Comment (iaguis) [reply .circleci/config.yml]:
```
$ kubectl -n=habitat-broker get pods -lapp=habitat-service-broker-habitat-service-broker
NAME                                                             READY     STATUS    RESTARTS   AGE
habitat-service-broker-habitat-service-broker-78956f5dc7-5fdw9   1/1       Running   0          2h
```

Comment (iaguis) [reply .circleci/config.yml]: https://github.com/kinvolk/habitat-service-broker/issues/29

**Pull request merged**